dist/src/main/arctic-bin/conf/config.yaml

```diff
@@ -91,3 +91,10 @@ containers:
 #      export.HADOOP_CONF_DIR: "/etc/hadoop/conf/"                   # Hadoop config dir
 #      export.HADOOP_USER_NAME: "hadoop"                             # Hadoop user submit on yarn
 #      export.FLINK_CONF_DIR: "/etc/hadoop/conf/"                    # Flink config dir
+
+#reporters:
```

Comment:
I don't think we need to add the configuration example right now as we have not implemented the whole functions.

Reply:
ok, i will remove it